features/step_definitions/networking.rb

```diff
@@ -790,3 +790,103 @@
   @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
   raise "the networkType is not #{network_type}" unless @result[:response] == network_type
 end
+
+Given /^the bridge interface named "([^"]*)" with address "([^"]*)" is added to the "([^"]*)" node$/ do |bridge_name,address,node_name|
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  @result = host.exec_admin("ip link add #{bridge_name} type bridge;ip address add #{address} dev #{bridge_name};ip link set up #{bridge_name}")
+  raise "Failed to add  bridge interface" unless @result[:success]
+end
+
+Given /^a DHCP service is configured for interface "([^"]*)" on "([^"]*)" node with address range and lease time as "([^"]*)"$/ do |br_inf,node_name,add_lease|
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  #Following will take dnsmasq backup and append curl contents to the dnsmasq config after
+  @result = host.exec_admin("cp /etc/dnsmasq.conf /etc/dnsmasq.conf.bak;curl https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/dnsmasq_for_testbridge.conf | sed s/testbr1/#{br_inf}/g | sed s/88.8.8.100,88.8.8.110,24h/#{add_lease}/g >> /etc/dnsmasq.conf;systemctl restart dnsmasq --now")
+  raise "Failed to configure DNS server" unless @result[:success]
+  step "10 seconds have passed"
+  if host.exec_admin("systemctl status dnsmasq")[:response].include? "running"
+    logger.info("DNS server is running fine")
+  else
+    raise "Failed to start DNS server. Check you cluster health manually"
+  end
+  raise "Failed to configure DNS server. Check your cluster health manually" unless @result[:success]
+end
+
+Given /^a DHCP service is deconfigured on the "([^"]*)" node$/ do |node_name|
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  #Copying original dnsmasq on to the modified one
+  @result = host.exec_admin("systemctl stop dnsmasq;cp /etc/dnsmasq.conf.bak /etc/dnsmasq.conf;systemctl restart dnsmasq --now")
+  raise "Failed to configure DNS server" unless @result[:success]
+  step "10 seconds have passed"
+  if host.exec_admin("systemctl status dnsmasq")[:response].include? "running"
+    logger.info("DNS server is running fine")
+  else
+    raise "Failed to bring your DNS server back to normal. Check you cluster health manually"
+  end
+  host.exec_admin("rm /etc/dnsmasq.conf.bak")
+end
+
+Given /^the vxlan tunnel name of node "([^"]*)" is stored in the#{OPT_SYM} clipboard$/ do |node_name,cb_name|
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  cb_name ||= "interface_name"
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  if @result[:success] then
+     networkType = @result[:response].strip
+  end
+  case networkType
+  when "OVNKubernetes"
+    @inf_name = host.exec_admin("ifconfig | egrep -o '^k8[^:]+'")
```

Comment:
@anuragthehatter does `@inf_name` needs to be an instance variable?  Can it be local variable instead?

Reply:
@pruan-rht. Thanks. Ya it needs to be local variable as scope is limited to this def only. Let me fix that